docs/docs/how-to/images-and-media/using-gatsby-plugin-image.md

```diff
@@ -0,0 +1,201 @@
+
+   ```jsx:title=src/components/dino.js
+   import { StaticImage } from "gatsby-plugin-image"
+
+   export function Dino() {
+     return <StaticImage src="../images/dino.png" alt="A dinosaur" />
+   }
+   ```
+
+   If you are using a remote image, pass the image URL in the `src` prop:
+
+   ```jsx:title=src/components/kitten.js
+   import { StaticImage } from "gatsby-plugin-image"
+
+   export function Kitten() {
+     return <StaticImage src="https://placekitten.com/800/600" alt="A kitten" />
+   }
+   ```
+
+   When you build your site, the `StaticImage` component will load the image from your filesystem or from the remote URL, and it will generate all the sizes and formats that you need to support a responsive image.
+
+   Because the image is loaded at build time, you cannot pass the filename in as a prop, or otherwise generate it outside of the component. It should either be a static string, or a local variable in the component's scope.
+
+   > **Important:** Remote images are downloaded and resized at build time. If the image is changed on the other server, it will not be updated on your site until you rebuild.
+
+3. **Configure the image.**
+
+   You configure the image by passing props to the `<StaticImage />` component. You can change the size and layout, as well as settings such as the type of placeholder used when lazy loading. There are also advanced image processing options available. You can find the full list of options in the API docs.
+
+   This component renders a 200px by 200px image of a dinosaur. Before loading it will have a blurred, low-resolution placeholder. It uses the `"fixed"` layout, which means the image does not resize with its container.
+
+   ```jsx:title=src/components/dino.js
+   import { StaticImage } from "gatsby-plugin-image"
+
+   export function Dino() {
+     return (
+       <StaticImage
+         src="../images/dino.png"
+         alt="A dinosaur"
+         placeholder="blurred"
+         layout="fixed"
+         width={200}
+         height={200}
+       />
+     )
+   }
+   ```
+
+   > To learn more about the different types of image layout, see the API docs.
+
+### Dynamic images
+
+If you need to have dynamic images (such as if they are coming from a CMS), you can load them via GraphQL and display them using the `GatsbyImage` component.
+
+1. **Add the image to your page query.**
+
+   Any GraphQL File object that includes an image will have a `childImageSharp` field that you can use to query the image data. The exact data structure will vary according to your data source, but the syntax is like this:
+
+   ```graphql:title=src/templates/blogpost.js
+   query {
+     blogPost(id: { eq: $Id }) {
+       title
+       body
+       # highlight-start
+       avatar {
+         childImageSharp {
+           gatsbyImageData(maxWidth: 200)
```

Comment:
```suggestion
           gatsbyImageData(layout: FIXED)
```
I'd either use no arguments in the initial example or make it layout since that's the most common. I know this opens a can of worms with sizes though.